.github/workflows/submodule_update.yml

```diff
@@ -0,0 +1,38 @@
+name: submodule_update
+
+on:
+  push:
+    branches: 
+      - 'master'
+      - '2.10'
+      - 'foxzi_update_submodule_workflow'
+
+jobs:
+  tarantool:
+    name: Update submodule in EE repository
+    runs-on: ubuntu-20.04-self-hosted
+    env:
+      SUBMODULE: 'tarantool'
+      REPOSITORY: 'tarantool/tarantool-ee'
+      FEATURE_BRANCH: 'TarantoolBot/update-tarantool-${{ github.ref_name }}'
+      CHECKOUT_BRANCH: master
+      PR_AGAINST_BRANCH: master
+      PR_TITLE: 'tarantool: bump submodule tarantool@${{ github.ref_name }}'
+      COMMIT_MESSAGE: |
+        tarantool: bump to new version
+      
+        NO_DOC=submodule update
+        NO_TEST=submodule update
+        NO_CHANGELOG=submodule update
+
+    steps:
+      - name: Create PR with submodule update
+        uses: tarantool/actions/update-submodule@master
+        with:
+          github_token: ${{ secrets.EE_UPDATE_SUBMODULE_TOKEN }}
+          repository: ${{ env.REPOSITORY }}
+          checkout_branch: ${{ env.CHECKOUT_BRANCH }}
+          feature_branch: ${{ env.FEATURE_BRANCH }}
+          pr_against_branch: ${{ env.PR_AGAINST_BRANCH }}
+          pr_title: ${{ env.PR_TITLE }}
```

Comment:
And change it to
```
${{ env.PR_TITLE_PREFIX }}${{ env.PR_TITLE }}
```

After my suggestions you have the following PR titles:
- `tarantool: bump to new version`
- `[2.10] tarantool: bump to new version`

Reply:
ok